crypto/crypto_int_test.go

```diff
@@ -84,19 +85,6 @@ func decodeHex(s string) []byte {
 	return d
 }
 
-// returns true if function called panic
-func shouldPanic(f func()) (didPanic bool) {
```

Comment:
This is dead code. Once `Decrypt()` panicked when the mac couldn't be verified, but that's long gone.